internal/handlers/dummy/msg_getfreemonitoringstatus.go

```diff
@@ -23,5 +23,5 @@ import (
 
 // MsgGetFreeMonitoringStatus implements HandlerInterface.
 func (h *Handler) MsgGetFreeMonitoringStatus(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, error) {
-	return common.MsgGetFreeMonitoringStatus(ctx, msg)
+	return common.MsgGetFreeMonitoringStatus(ctx, msg, nil)
```

Comment:
It looks a bit controversial to me that we check for state not being `nil` in `MsgGetFreeMonitoringStatus` but here we set it to `nil` on purpose. I understand the idea - this is just a stub, but I wonder if we should leave a comment or something.

Reply:
Is it good now? :)